cmd/wallet-sdk-gomobile/api/diddocresolution.go

```diff
@@ -60,7 +60,7 @@ func (d *DIDDocResolution) ID() (string, error) {
 
 // AssertionMethod returns did assertion verification method.
 func (d *DIDDocResolution) AssertionMethod() (*VerificationMethod, error) {
-	didDocResolutionParsed, err := diddoc.ParseDocumentResolution(d.Content)
+	didDocResolutionParsed, err := diddoc.ParseDocumentResolution([]byte(d.Content))
```

Comment:
(out of scope for this PR, but ideally we should change this so that we don't have to reparse all the time - similar to the VerifiableCredential type)

Reply:
@rolsonquadras

Reply:
yes, definitely